### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by wademix**
Capture

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -44,12 +47,6 @@
       case '/interns/test' :
           require __DIR__ . '/views/interns/test.php';
           break;
-      case '/interns/david' :
```
Comment (Victor-Ugwueze) [inline index.php]: why did you remove these lines of code?

**Comment by wademix** [comment]
Mistake.. Is that my only problem..  Can I resend?
